scripts/apps/monitoring/services/CardsService.js

```diff
@@ -67,7 +67,13 @@ export function CardsService(api, search, session, desks, config) {
 
         default:
             if (!_.isNil(card.singleViewType) && card.singleViewType === 'desk') {
-                query.filter({term: {'task.desk': card.deskId}});
+                query.filter({or: [
+                        {terms: {'marked_desks.desk_id': [card.deskId]}},
```

Comment:
a minor one. change this filter to `term` instead of `terms` and outcome will be the same.